experimental/e2e/storage/har.go

```diff
@@ -18,9 +18,64 @@ import (
 	"github.com/grafana/grafana-plugin-sdk-go/experimental/e2e/utils"
 )
 
+type file struct {
+	sync.RWMutex
+	Path string
```

Comment:
This can be unexported, no?

Reply:
good catch, this is an artifact of a previous iteration

Reply:
unexported in [a04c90c](https://github.com/grafana/grafana-plugin-sdk-go/pull/491/commits/a04c90c45484a0c7046d177b47ad6f431045e072)